datumaro/plugins/synthetic_data/synthetic_background.txt

```diff
@@ -0,0 +1 @@
5, 110.17371676, 131.66609497, 102.72913733, 113.94528838, 128.46481906,  81.72760135, 110.13304943, 134.23034765,  88.7570118,  115.94842629, 143.85741514,  90.54364748, 108.77935253, 132.10548753,  86.69857458, 117.06207974, 143.24318599,  93.25191696, 116.8354967,  139.34347813,  90.34648234, 121.82992859, 143.32214135,  89.68316398, 116.74557908, 102.14592523,  88.84627112, 120.66835867, 109.64541671, 105.56420123, 139.43363856, 144.53888846,  82.2065151,  116.68337096, 115.95974984,  56.78486312, 113.70027687, 140.9087729,  81.65051399, 114.54978331, 130.79190633,  64.56654323, 107.05065546, 146.38669212,  86.67132459, 117.9284256,  118.16428023,  84.41742206, 107.15938828, 111.08075375,  72.64669476, 118.47903683, 125.36536822,  99.09798257, 112.09365318, 110.84019321,  69.99219619,  93.44804385, 106.59032831, 101.77054384, 146.88977724, 144.38213881,  86.96338136, 109.5160261,  151.39501042,  50.9497511,  103.17187821, 150.57970488,  75.21457287, 121.30832859, 138.06993947,  90.86574104, 119.10118793, 133.7632379,  76.04992864, 103.96868816, 112.49819041,  66.91270748, 116.2897105,  133.83198743,  69.99665178, 109.33478707, 107.29829018, 103.03329586, 135.40640106, 135.82532425,  67.89173184,  86.24811818, 126.08736545, 101.84951875, 122.0955736,  125.37005846,  97.21195821, 128.4305905,  149.75465887,  91.482291,   105.95839098, 124.44834602,  95.00323812, 114.4004559,  133.52674445,  82.61139597, 106.77707163, 137.44438891,  78.62984882, 106.25381385, 138.47251182,  83.45551806, 113.13841268, 136.43813737,  93.26349137, 113.78889717, 131.84397581,  87.62114582, 100.1495592,  118.43652661,  93.02587456, 107.64662456, 123.74482832, 108.72062844, 120.41733612, 134.47839148,  81.90065444, 100.24442356, 119.59022445, 145.66667831, 137.86594597, 121.58729982,  93.22558525, 101.55920145, 106.78988974, 117.52006598, 125.38797884, 127.41080753, 121.09277082, 116.02339168,  76.90280136, 140.39008498, 136.41892422, 129.96746646, 126.84253424, 124.19296287, 115.44630753, 145.15557007, 134.16390633, 116.95433047, 148.50742464, 141.2782068,  131.37579313, 119.47619251, 121.91313125, 120.02422501,  99.83548534, 107.7379105,   94.83031186, 123.0431712,  115.57177483, 109.61376472,  98.03354706, 117.15164077, 122.26742621,  88.72856927, 103.15021271, 115.57492771, 135.44214441, 114.1796108,   54.81603952,  87.11776744, 140.57112921, 131.75653179,  76.14453861, 101.02743176, 109.87231679, 43.27891438, 99.8487709,  84.32860689,  73.18576994, 115.44833992, 139.85550173,  82.86026332, 110.3486101,  112.55075207,  83.91833122, 104.11213943, 115.29845265,  75.78574139, 103.32525733, 120.81529803, 65.46688133, 86.99320303, 97.93580678,  66.40035271,  88.72632415, 107.13842382,  77.36862105,  97.47567595, 115.09720107,  73.55448181,  94.33005634, 104.46628367, 66.70040204, 83.72144382, 93.05657139,  85.36132023, 103.81117533, 111.53193086,  93.40080101, 116.77883711, 124.83443006,  73.95684449, 118.50754695, 131.90257437, 102.32897641, 115.00227251, 126.53089914
```

Comment:
What are these numbers? Can we round the values to 2 floating point digits? Can we format them better?

Reply:
I generated this file using the mean value by class for Imagenet images per channel. I regenerated file with int values per channel.

Reply:
I think you should add the script that you used to create this file and add a comment to the file pointing to it (or a comment with a URL if the script is already uploaded somewhere else).

Reply:
So how about that script? You can put it in the recently added `dev` directory.